validation/genesis/validation-inputs/16481/meta.toml

```diff
@@ -0,0 +1,4 @@
+genesis_creation_commit = "f8143c8cbc4cc0c83922c53f17a1e47280673485"
+node_version = "18.12.1"
+monorepo_build_command = "pnpm"
+genesis_creation_command = "opnode2"
```

Comment:
I was able to get this to pass locally by changing this value `opnode2` --> `forge1`. Please push that change and we should see the ci test suite pass as a result

Reply:
Hey thanks for pointing out the issue. Seems to pass validation on local:

`just validate 16481`:
![Screenshot 2024-12-04 at 1 28 57 PM](https://github.com/user-attachments/assets/cd1be21e-482f-46a2-bf8b-c1f4b6237f61)

`just validate-genesis-allocs 16481`:
![Screenshot 2024-12-04 at 1 29 09 PM](https://github.com/user-attachments/assets/1e73f1d4-d495-4554-ab8f-2ad6768d2590)